meta-oe/recipes-test/trompeloeil/trompeloeil_v47.bb

```diff
@@ -0,0 +1,16 @@
+DESCRIPTION = "A popular statically typed mocking framework for C++14 and later"
+AUTHOR = "Björn Fahller, Andrew Paxie and others"
```

Comment:
We have removed `AUTHOR` field see https://git.yoctoproject.org/poky/commit/?id=16dff2be01e8fc73e0057e3795319ea01fdeae5c

so please remove it.

Reply:
Done. Thx for the hint!

Reply:
please squash both commits into one and update the PR please

Reply:
Check!